pypeit/fluxspec.py

```diff
@@ -283,7 +279,14 @@ def show_sensfunc(self):
         #mag_func = utils.func_val(self.sens_dict['c'], wave, self.sens_dict['func'])
         #sens = 10.0**(0.4*mag_func)
         # Plot
-        debugger.plot1d(self.sens_dict['wave'], self.sens_dict['sensfunc'], xlbl='Wavelength', ylbl='Sensitivity Function')
+        if self.spectrograph.pypeline == 'Echelle':
```

Comment:
Can you please get the old debugger out of here, it is deprecated.

Reply:
The old debugger is not deprecated.
Only its wrapper to `pdb`.  Removing that now.